### ISSUE TITLE
Accessibility: Question rating not keyboard-accessible

### ISSUE THREAD

**Issue by HuongNV13** [frankkoch/moodle-mod_studentquiz#129]
Hi Frank,
We found an issue related to accessibility as below:

-------------------------------------------------------------------
The 'Rate' feature is not accessible for keyboard or screen reader users.

This element is not tabbable or recognisable by screen readers because each star is an HTML 'span' element with no helpful accessibility information.

Make the 'Rate' stars into radio buttons with labels and accessible descriptions. Visually hide the radio buttons and text inside the labels and use CSS/SVG/JavaScript to create the stars, indicate selected stars and provide output e.g. 2 Stars selected. See Web Accessibility Tutorials Custom Controls http://www.w3.org/WAI/tutorials/forms/custom-controls/.

Thanks,
